app/assets/stylesheets/_account.scss

```diff
@@ -15,9 +14,15 @@
     display: none;
   }
 
+  .new-account {
+    display: flex;
```

Comment:
I usually see `@include display(flex)` and/or `@include flex(1)` rather than plain `display: flex`. Thoughts?